src/main/java/org/eolang/speco/Speco.java

```diff
@@ -125,7 +125,7 @@ public static XML applyTrain(final XML xml) {
      * @param input XML input
      * @return XML
      */
-    private static XML getParsedXml(final XML input) {
+    public static XML getParsedXml(final XML input) {
```

Comment:
@advasileva Can we rename that method to `parse`, or `specify`, or something in one word?
Also I would highly recommend you to read that [article](https://www.yegor256.com/2018/08/22/builders-and-manipulators.html).

Reply:
> If the method returns something, it has to be a noun.

If we are guided by the article, then we cannot rename this imperative method to `parse`, `specify` or another verb, because it returns a new object. At the same time, naming it `xml()` is even worse. 

Apparently, we will have to change the design of the code to make it more object-oriented. In the last commit, I removed this method and distributed the functionality to the call locations. Code duplication is minimal (since `Train` is already used there) and allows me to make the test less dependent on the `Speco` implementation. Another solution would be to turn this method into a separate decorator class, but I think this is overengineering.